bindata/manifests/daemon/daemonset.yaml

```diff
@@ -71,6 +71,28 @@ spec:
           preStop:
             exec:
               command: ["/bindata/scripts/clean-k8s-services.sh"]
+      - name: sriov-cni
+        image: {{.SRIOVCNIImage}}
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            cpu: 10m
+            memory: 10Mi
+        volumeMounts:
+        - name: cnibin
+          mountPath: /host/opt/cni/bin
+      - name: sriov-infiniband-cni
+        image: {{.SRIOVInfiniBandCNIImage}}
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            cpu: 10m
+            memory: 10Mi
+        volumeMounts:
+        - name: cnibin
+          mountPath: /host/opt/cni/bin
```

Comment:
Did you run `make manifests bundle` too?

Reply:
I did not. 

Here's what I get when I run `make manifests bundle`:

```
[jerpeter@fedora sriov-network-operator-upstream]$ make manifests bundle
/home/jerpeter/src/sriov-network-operator-upstream/bin/controller-gen "crd:crdVersions={v1}" webhook paths="./..." output:crd:artifacts:config=./config/crd/bases
cp ./config/crd/bases/* ./deployment/sriov-network-operator/crds/
operator-sdk generate kustomize manifests --interactive=false -q
cd config/manager && /home/jerpeter/src/sriov-network-operator-upstream/bin/kustomize edit set image controller=controller:latest
/home/jerpeter/src/sriov-network-operator-upstream/bin/kustomize build config/manifests | operator-sdk generate bundle -q --overwrite --version 4.7.0
Error: accumulating resources: accumulation err='accumulating resources from '../samples': '/home/jerpeter/src/sriov-network-operator-upstream/config/samples' must resolve to a file': couldn't make target for path '/home/jerpeter/src/sriov-network-operator-upstream/config/samples': unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization' in directory '/home/jerpeter/src/sriov-network-operator-upstream/config/samples'
INFO[0000] Creating bundle.Dockerfile
INFO[0000] Creating bundle/metadata/annotations.yaml
INFO[0000] Bundle metadata generated suceessfully
make: *** [Makefile:173: bundle] Error 1

[jerpeter@fedora sriov-network-operator-upstream]$ git status
On branch support-graceful-shutdown
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   bundle.Dockerfile
	modified:   config/manager/kustomization.yaml

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	bundle/
	config/manifests/

no changes added to commit (use "git add" and/or "git commit -a")

[jerpeter@fedora sriov-network-operator-upstream]$ git diff
diff --git a/bundle.Dockerfile b/bundle.Dockerfile
index bf295a04..7b73150d 100644
--- a/bundle.Dockerfile
+++ b/bundle.Dockerfile
@@ -1,16 +1,20 @@
 FROM scratch

+# Core bundle labels.
 LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=sriov-network-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
-LABEL operators.operatorframework.io.bundle.channel.default.v1=
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.0.1
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.12.0+git
 LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
-LABEL operators.operatorframework.io.metrics.project_layout=go.kubebuilder.io/v2
-LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
+LABEL operators.operatorframework.io.metrics.project_layout=go.kubebuilder.io/v3
+
+# Labels for testing.
 LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
+LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/

+# Copy files to locations specified by labels.
 COPY bundle/manifests /manifests/
 COPY bundle/metadata /metadata/
+COPY bundle/tests/scorecard /tests/scorecard/
diff --git a/config/manager/kustomization.yaml b/config/manager/kustomization.yaml
index 2bcd3eea..5e793dd1 100644
--- a/config/manager/kustomization.yaml
+++ b/config/manager/kustomization.yaml
@@ -5,6 +5,12 @@ generatorOptions:
   disableNameSuffixHash: true

 configMapGenerator:
-- name: manager-config
-  files:
+- files:
   - controller_manager_config.yaml
+  name: manager-config
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: controller
+  newTag: latest

```

Reply:
please commit that too in a separate commit.

Please fix 

`make: *** [Makefile:173: bundle] Error 1` first

Reply:
I thought I had resolved this and pushed a branch, but that doesn't appear to be the case. Today, I'm getting the same error running `make manifests bundle` from the head of the current master branch (without any of my changes):

```
[jerpeter@fedora sriov-network-operator-upstream]$ git rev-parse --short HEAD
990648ee
[jerpeter@fedora sriov-network-operator-upstream]$ git reset --hard upstream/master
HEAD is now at 990648ee Merge pull request #451 from zeeke/exclude-topology-conformance-test
[jerpeter@fedora sriov-network-operator-upstream]$ git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
[jerpeter@fedora sriov-network-operator-upstream]$ make manifests bundle
/home/jerpeter/src/sriov-network-operator-upstream/bin/controller-gen "crd:crdVersions={v1}" webhook paths="./..." output:crd:artifacts:config=./config/crd/bases
cp ./config/crd/bases/* ./deployment/sriov-network-operator/crds/
operator-sdk generate kustomize manifests --interactive=false -q
cd config/manager && /home/jerpeter/src/sriov-network-operator-upstream/bin/kustomize edit set image controller=controller:latest
/home/jerpeter/src/sriov-network-operator-upstream/bin/kustomize build config/manifests | operator-sdk generate bundle -q --overwrite --version 4.7.0
Error: unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization' in directory '/home/jerpeter/src/sriov-network-operator-upstream/config/manifests'
INFO[0000] Creating bundle.Dockerfile
INFO[0000] Creating bundle/metadata/annotations.yaml
INFO[0000] Bundle metadata generated successfully
make: *** [Makefile:180: bundle] Error 1
[jerpeter@fedora sriov-network-operator-upstream]$ operator-sdk version
operator-sdk version: "v1.30.0", commit: "b794fe909abc1affa1f28cfb75ceaf3bf79187e6", kubernetes version: "v1.26.0", go version: "go1.20.5", GOOS: "linux", GOARCH: "amd64"
```

Any suggestions?

Reply:
I've investigated this a bit, I think we don't support csv in upstream. I like other's chime in too but I think we should remove `make bundle` completely from upstream (only).

Reply:
@SchSeba We need to get this one merged. Please chime in wrt to csv u/s support. I think we should remove `make bundle` to avoid confusion. Then I'm ok to merge this.

Reply:
> I think we should remove make bundle to avoid confusion.

Is that going to happen in another PR?

Reply:
Yes, @jerpeter1 please prep a PR that does exactly that. We can get that one merged first.